test/Microsoft.Azure.Devices.Edge.Test.Common/linux/PackageManagement.cs

```diff
@@ -62,7 +62,35 @@ public string[] GetInstallCommandsFromLocal(string path)
 
         public string[] GetInstallCommandsFromMicrosoftProd(Option<Uri> proxy)
         {
-            throw new NotImplementedException("aziot-edge and aziot-identity-service currently aren't available in package repos");
+            // we really support only two options for now.
+            string repository = this.os.ToLower() switch
+            {
+                "ubuntu" => $"https://packages.microsoft.com/config/ubuntu/18.04/multiarch/prod.list",
+                "debian" => $"https://packages.microsoft.com/config/debian/stretch/multiarch/prod.list",
```

Comment:
You don't need '$' for these strings at the moment.

Reply:
And a few others in this function. I'm surprised the style checking didn't flag this.